src/snapred/backend/dao/state/FocusGroup.py

```diff
@@ -12,3 +12,26 @@ class FocusGroup(BaseModel):
     dMax: List[float]
     dMin: List[float]
     definition: str
+
+    @root_validator(pre=True, allow_reuse=True)
```

Comment:
unfortunately these values will be going away in favor of pixelgroupingparameters, just a heads up.

Reply:
That will change a lot.  How would that square with the other PR #154

Reply:
I'm not sure these values are actually being used, it should be noted that they are about equivalent to pixelgroupingparameters.  The rational for their removal is the separation of static focusgroup data like name and definitionfile from the calculated values.

Reply:
I can guarantee these values are actually being used, inside the diffraction calibration process.  Changing them will require some additional changes to this workflow.